docs/wiki/The CI environment - ReadMe status table update.md

```diff
@@ -0,0 +1,55 @@
+Use these scripts to update the diverse module and pipeline status tables (such as 'Available Resource Modules').
```

Comment:
Is this script intended to be used as a utility outside of the pipeline using it? If not, I'd suggest to remove this page and, if useful, integrate the high level design content in the CI environment pipeline design page

Reply:
Yes and no - but probably not. If you'd agree I'd move some of the content then to the pipeline files themselves

Reply:
Agree, let's do that

Reply:
done